api/test/crossReference.test.js

```diff
@@ -0,0 +1,19 @@
+import fetch from "node-fetch";
+
+describe("cross reference", () => {
+  test("should return more than one component", async () => {
+    const res = await fetch(`${API_BASE}/external-db/HMR%202.0/HMR_1157`);
+
+    const { components } = await res.json();
+    expect(components.length).toBeGreaterThan(1);
+  });
+
+  test("should return a TSV file for a given data source", async () => {
+    const res = await fetch(`${API_BASE}/external-db/Recon3D/RE3335C`);
+
+    const {
+      externalDb: { url },
+    } = await res.json();
+    expect(url.length).toBeGreaterThan(1);
```

Comment:
I think it would be possible to make an http request to the url and see if the response is successful. I'm not sure if it would be too beneficial though because it only validates one of many possible URLs we have in the database. The intention here was to make sure that an external reference (to Recon3D as opposed to one of our own cross references like HMR) could have an URL.